Azumuta/eslint-plugin-azumuta
# Fix multiline align imports

fixes DEV-1377

Warning: with these changes, the imported functions etc are sorted, we might also try to make the imports more compact, ...


To test:
npm un eslint-plugin-azumuta
npm i ../eslint-plugin-azumuta

Should apply this renaming in one commit to avoid pull requests with formatting changes
